tests/python/pytorch/nn/test_nn.py

```diff
@@ -2551,10 +2551,15 @@ def test_SpatialEncoder(max_dist, num_kernels, num_heads):
     g2 = dgl.graph(
         (th.tensor([0, 1, 2, 3, 2, 5]), th.tensor([1, 2, 3, 4, 0, 3]))
     ).to(dev)
-    bg = dgl.batch([g1, g2])
-    ndata = th.rand(bg.num_nodes(), 3).to(dev)
-    num_nodes = bg.num_nodes()
-    node_type = th.randint(0, 512, (num_nodes,)).to(dev)
+    coord = th.rand(2, 6, 3).to(dev)
```

Comment:
Try avoiding using hard-coded numbers and using more semantically meaningful variable names.

Reply:
fixed